packages/ListBoxWithTags/src/ListBoxWithTags.js

```diff
@@ -129,7 +129,7 @@ export default function ListBoxWithTags(props) {
 
   return (
     <div ref={refDivRoot}>
-      <ListBox ref={refListBox} isMulti size={validSize} onChange={handleChange} {...moreProps}>
+      <ListBox ref={refListBox} isMulti hasTag size={validSize} onChange={handleChange} {...moreProps}>
```

Comment:
if we changed the src files in ListBoxWithTags, we need to add this component to the changeset too uwu